test/apps/iodemo/ucx_wrapper.cc

```diff
@@ -111,10 +113,18 @@ void UcxContext::UcxDisconnectCallback::operator()(ucs_status_t status)
     delete this;
 }
 
-UcxContext::UcxContext(size_t iomsg_size, double connect_timeout, bool use_am) :
-    _context(NULL), _worker(NULL), _listener(NULL), _iomsg_recv_request(NULL),
-    _iomsg_buffer(iomsg_size, '\0'), _connect_timeout(connect_timeout),
-    _use_am(use_am)
+UcxContext::UcxContext(size_t iomsg_size, double connect_timeout, bool use_am,
+                       bool use_epoll) :
+    _context(NULL),
+    _worker(NULL),
+    _listener(NULL),
+    _iomsg_recv_request(NULL),
+    _iomsg_buffer(iomsg_size, '\0'),
+    _connect_timeout(connect_timeout),
+    _use_am(use_am),
+    _use_epoll(use_epoll),
+    _worker_fd(0),
+    _epoll_fd(-1)
```

Comment:
maybe rever to the previous way so each line will have multiple vars?

Reply:
Done